### ISSUE TITLE
metrics/display: iterable job status

### ISSUE THREAD

**Issue by bf4** [sidekiq/sidekiq#6537]
Ruby version: 3.3
Rails version: 7.0
Sidekiq Pro: 7.2.4

I'm running our first iterable job and was surprised that the web ui doesn't have a way to show the iteration progress.

We're using the array_enumerator and I was hoping to see something like "running on item #{item}".

I didn't see a way to see this information in the released code or open issues/pull requests.

In the same vein, I see it the web ui has "Started". I hope hoped to see something like "iteration job start" and "current iteration start".

A tab like for batches would be a good affordance, I think.

**Comment by mperham** [comment]
I'm not convinced a UI like that would be used in practice. What's the usecase beyond novelty?

**Comment by bf4** [comment]
I have a worker "OrdertImportWorker" which specifies it should iterate over various import tasks "import_recent_orders", "import_related_tasks",  etc.  

When it's running I'd like to get a sense of how far along it is.  Each iteration is relatively slow, maybe 10 minutes, so I can estimate that 20 minutes in, perhaps it's on the second item, but I don't really know.  Seeing that 1. the job has been running for 20 minutes 2. it's on iteration two  and 3. it's been on this iteration for 5 minutes would help me understand, at a glance, how much progress it's made. As it is, an iterable job looks exactly like a very long running job. Since it's not exactly, for me, it would be helpful to see it.

I see some form of this data is available

```ruby
      def fetch_previous_iteration_state
        state = Sidekiq.redis { |conn| conn.hgetall(iteration_key) }

        unless state.empty?
          @_executions = state["ex"].to_i
          @_cursor = Sidekiq.load_json(state["c"])
          @_runtime = state["rt"].to_f
        end
      end
```

it looks like the "Start" field in the web ui comes from `work.run_at`. Which, I think means, you could have something like `job = work.job; if job.include?(Sidekiq::IterableJob); job.fetch_previous_iteration_state["ex"]` though I see now that would be inefficient

**Comment by mperham** [comment]
We're starting to get a lot of tabs in the UI so I'd rather not add another tab for this data. It's possible I can add it to the Jobs table in Busy. I'm not sure how useful the display can be; I can show things like Total Runtime but something like the cursor is opaque to Sidekiq and may not be useful to display.

**Comment by joevandyk** [comment]
I was just about to ask for this exact same thing.  Adding it to the jobs view would be great. Showing the cursor and any other available state would be great.

**Comment by mperham** [comment]
PRs welcome if someone wants to work on this and supply a screenshot for further discussion.

**Comment by joevandyk** [comment]
thoughts? there's a new 'context' column in the busy view.  https://github.com/sidekiq/sidekiq/pull/6787

<img width="3562" height="2502" alt="Image" src="https://github.com/user-attachments/assets/a830a464-b3d5-4274-81f5-8a1ad6878360" />

**Comment by mperham** [comment]
Adding another column for something that is likely to be little used (iteration is still very new) seems questionable. That real estate is very precious. What if we combine started at and context into one status column, two lines maybe?

**Comment by joevandyk** [comment]
@mperham like this? 

<img width="764" height="527" alt="Image" src="https://github.com/user-attachments/assets/c51f0bd7-aa97-42a1-8d5a-80434357a6d4" />

**Comment by mperham** [comment]
Exactly. I'm not a designer at all, but that seems like the best approach that I can determine. If the job is not iterable, the second line will not be there.

**Comment by joevandyk** [comment]
@mperham  thoughts?

<img width="795" height="496" alt="Image" src="https://github.com/user-attachments/assets/da899900-4b1b-44a9-82d4-44ad4e6dbecb" />

**Comment by mperham** [comment]
I think that looks good enough but I would be open to further polish. Adding `Cursor: ` to the front seems redundant? And requires a new localized string.
